### ISSUE TITLE
Convert image formats to rgba32 like hardware does

### ISSUE THREAD

**Issue by Dragorn421** [NEstelami/ZAPD#237]
As noticed by mdta on the zeldaret discord ( https://discord.com/channels/688807550715560050/688851317593997489/942010735922393129 ), the pngs extracted by ZAPD are a bit off. For example IA8 images where they should be fully opaque are only ~97% opaque

This may only be an issue with IA8 or on all formats I'm not sure.
Maybe fixing it will need tweaking the png to .inc conversion too to preserve matching, I don't know either.

--------------------

For example here's how angrylion (a highly accurate video plugin) converts IA8 to rgba32:

https://github.com/ata4/angrylion-rdp-plus/blob/6660e93acc94506933238c9a593494684d0d3bde/src/core/n64video/rdp/tmem.c#L332-L337
```c
            i = p & 0xf0;
            i |= (i >> 4);
            color->r = i;
            color->g = i;
            color->b = i;
            color->a = ((p & 0xf) << 4) | (p & 0xf);
```

which indeed doesn't match how ZAPD converts IA8:

https://github.com/zeldaret/ZAPD/blob/14d2a734b3fdddf2091629edc59db94ab5548f3e/ZAPD/ZTexture.cpp#L262-L263

**Comment by mdtauk** [comment]
I believe this also affects ia4 files, as the white colour from the image extracted via Texture64 (bottom image) is off white (#e0e0e0) with the same texture exported via ZAPD and the make process (top image)

![image](https://user-images.githubusercontent.com/7389110/153709764-0a5196c0-0ba5-4e73-bf26-6c35ef80b7a3.png)

**Comment by Dragorn421** [comment]
As an additional reference

The conversion from every format to rgba32 is actually *in the manual*

Chapter 13 - Texture Mapping
13.8.2 Texel Formatting
http://n64devkit.square7.ch/pro-man/pro13/13-08.htm#02
> In the RDP graphics pipeline, most operations are done on 8-bit-per-component RGBA pixels. After looking up the texels, the texture unit converts them into the 32-bit RGBA format. Table 2 describes how each type is converted. The format for beatified descriptions is [MSB:LSB] where MSB is the most significant bit and LSB is the least significant bit. Bit fields are grouped together in braces {} with the most significant field on the left and the least significant field on the right. 
> ![image](https://user-images.githubusercontent.com/5306619/205668757-a2c535ca-0212-42fe-80c1-7bd61a727a12.png)

**Comment by NEstelami** [comment]
Resolved by PR #269.
